strings-csharp/HowToAddSeparatorsInThousandsPlaceForANumber/HowToAddSeparatorsInThousandsPlaceForANumber/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Globalization;
+
+// String.Format() Method
+
+var balance = 2154002.535;
+
+var balanceMessage = String.Format("Your account balance is: {0}", balance);
+
+Console.WriteLine(balanceMessage);
+
+var englishCultureInfo = new CultureInfo("en-US");
+
+var balanceMessageWithCultureInfo = String.Format("Your account balance is: {0}", balance, englishCultureInfo);
+
+Console.WriteLine(balanceMessageWithCultureInfo);
+
+var balanceMessageWithSeparators = String.Format("Your account balance is: {0:n}", balance, englishCultureInfo);
+
+Console.WriteLine(balanceMessageWithSeparators);
+
+var balanceMessageWithFourDigits = String.Format("Your account balance is: {0:n4}", balance, englishCultureInfo);
+
+Console.WriteLine(balanceMessageWithFourDigits);
+
+var balanceMessageWithoutDecimals = String.Format("Your account balance is: {0:n0}", balance, englishCultureInfo);
+
+Console.WriteLine(balanceMessageWithoutDecimals);
+
+// The ToString() method
+
```

Comment:
@biagianina 
I think to align with other methods, it will look good to make changes to this section
e.g 
`var formattedBalance = "Your account balance is: " + balance.ToString("n");`
`Console.WriteLine(formattedBalance);`

Reply:
@biagianina 
I think we can safely remove the comments. And also the comments in the tests as well.

Reply:
Done :)